lib/sorcery/providers/linkedin.rb

```diff
@@ -9,25 +9,26 @@ module Providers
     class Linkedin < Base
       include Protocols::Oauth2
 
-      attr_accessor :auth_url, :scope, :token_url, :user_info_url
+      attr_accessor :auth_url, :scope, :token_url, :user_info_url, :email_info_url
```

Comment:
Do we actually need this? [The docs](https://developer.linkedin.com/docs/fields/basic-profile) seem to indicate all you need is to include `r_emailaddress` in the scope.